### ISSUE TITLE
More issues with DocCommentSniff

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#275]
In the following code the absence of short description is reported (code `MissingShort`), however for inline block comments, like `@var` or `@type` such thing isn't a required part.

Also some issue with missing blank lines is also reported, but I have no idea why it's reported (code `SpacingAfter`)

``` php
/** @var NodeElement $content_box_title */
$content_box_title = array_shift($content_box_titles);
```

`phpcs` output:

```
--------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 2 LINES
--------------------------------------------------------------------------------
  2 | ERROR | [x] There must be one blank line after the namespace declaration
    |       |     (PSR2.Namespaces.NamespaceDeclaration.BlankLineAfter)
 25 | ERROR | [ ] Missing short description in doc comment
    |       |     (CodingStandard.Commenting.DocCommentWrapper.MissingShort)
 25 | ERROR | [x] Additional blank lines found at end of doc comment
    |       |     (CodingStandard.Commenting.DocCommentWrapper.SpacingAfter)
--------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------
```

This is the file: https://gist.github.com/aik099/fc192ba0b9148002421d

I think we seriously need to address this within sniff itself, because more and more reported issues doesn't make any sense in context of inline block comments that have `@var` or `@type` in them.

Related to #258
